azure-pipelines.yml

```
@@ -0,0 +1,75 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+pool:
+  name: 'Continuous Integration 02 - SSD - 160ACU'
+  demands: 
+  - msbuild
+  - visualstudio
+
+steps:
+- task: gittools.gitversion.gitversion-task.GitVersion@3
```

If you're interested in using [GitVerison Mainline Development](https://gitversion.readthedocs.io/en/latest/reference/mainline-development/) you can update this task to version 4:
```suggestion
- task: gittools.gitversion.gitversion-task.GitVersion@4
```
then add this file to the root of the repository:
https://github.com/SkillsFundingAgency/das-apply-service/blob/master/GitVersion.yml

Up to you